.github/workflows/gradle.yml

```diff
@@ -77,7 +66,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
         with:
-          submodules: recursive
+          submodules: false
+
+      - name: Get Submodules
+        run: git submodule update --init --recursive linea-constraints
```

Comment:
Does this means it will update the submodule to the latest master in every PR ? If yes, it shouldn't be the case, while debugging, we need to point to a specific constraint commit

Reply:
My understanding is that it will update the submodule to the version specified in the `.gitmodules` file